.github/workflows/php7.2.yml

```diff
@@ -4,7 +4,7 @@ on: ["push", "pull_request"]
 
 env:
   doctrine_website_github_http_token: "${{ github.token }}"
-  PHP_VERSION: "7.2"
+  PHP_VERSION: "7.4"
```

Comment:
Hmmmm… the name of that file contains the php version

Reply:
Yeah, a stupid decision. I'll rename it to `continuous-integration.yml`.